pkg/commands/compute/serve.go

```diff
@@ -168,6 +168,16 @@ func getViceroy(progress text.Progress, out io.Writer, versioner update.Versione
 		}
 	}
 
+	// G302 (CWE-276): Expect file permissions to be 0600 or less
+	// gosec flagged this:
+	// Disabling as the file was not executable without it and we need all users
+	// to be able to execute the binary.
+	/* #nosec */
+	err = os.Chmod(bin, 0777)
```

Comment:
Why not 0555?

Reply:
For now I'm just keeping it consistent to what we've used elsewhere. I'll make a note to revisit this as 555 would probably be fine.